cmd/cri-containerd/cri_containerd.go

```diff
@@ -93,15 +94,19 @@ func main() {
 		}
 		validateConfig(o)
 
-		glog.V(0).Infof("Run cri-containerd %+v", o)
+		if err := setLogLevel(o.LogLevel); err != nil {
+			return fmt.Errorf("failed to set log level: %v", err)
+		}
+
+		logrus.Infof("Run cri-containerd %+v", o)
```

Comment:
before setLogLevel？

Reply:
We want to make sure `setLogLevel` also works for this log line, right?